### ISSUE TITLE
Feature: Prefill URL and Username in additional data on insert/generate dialogues

### ISSUE THREAD

**Issue by BenjaminDebeerst** [passff/passff#380]
Following my PR #379, I think it would be a nice feature to have additional info pre-filled. I think there are many options and probably also some opinions, so I figured I should open an issue first to discuss before starting to work on it.

Proposal:

* When opening the "New" dialogue, the "additional info" textbox is pre-filled with data (both for insert and generate dialogue).
* Two lines should be pre-filled, if corresponding data can be found: Username and URL.
* The data pre-filled is sourced from the tab opened when opening that dialogue. The username is fetched in the same way as filling a form would be finding the input to fill. The URL is extracted from the current tab's URL.
* The names for the two fields shall be taken from the plugin's configured possible values for those fields, e.g. by picking the first out of the list. By default it would be `login` and `url`.
* This functionality should be opt-in; separate toggles for each of the pre-filled fields.

Considerations:

* Pre-filling a URL might be a bad idea: Auto-filling data into websites is a possible leak of information. If a password is created using passff with an unrelated browser tab open, the then created secret is no associated with an unrelated URL. Users must be aware of this and may need to manually edit the `additional_info` textbox. This is why I think the feature should be opt-in.
* Pre-filling the username might be just a bad idea:
  * In many cases, a username for the website might not be found, because there is no such form field present on typical "change password"  pages.
  * When multiple username fields are found, there might be conflict over which on to use. Should we just pick the first?
  * When reading data from a website to pass that string into an invocation of `pass`, this must be well escaped, otherwise a malicious website might be able to execute code and e.g. extract passwords

  As result of this consideration might be to abandon pre-filling a username altogether. 

As you can see, I am myself not totally convinced that this is a great idea. Let me know what you think.

**Comment by tuxor1337** [comment]
Yeah, sounds like a reasonable enhancement, at least as an opt-in preference.

**Comment by Toromtomtom** [comment]
If this would be implemented, I would gladly use it. The username is often an Email address, so maybe a default Email address could be a configuration option.
